ita-social-projects/GreenCityUBS
# Order History partially paid fixed `READY_FOR_REVIEW`

# Order History partially paid fixed PR `READY_FOR_REVIEW`

## Summary Of Changes :fire:
The "Замовлення Частково оплачено" event didn't appear in the "Історія замовлення" order work log after the payment was added by the administrator and the actual amount of the order was greater than the paid amount.

## Issue Link 🔗 
[Git Issue Card #4507](https://github.com/ita-social-projects/GreenCity/issues/4507)

## Added
- Added a validation in buildPaymentEntity on the Service Implement, to check if the order is fully or partially paid
- Added the ORDER_HALF_PAID description

## How to test
This could be tested with Swagger also
1. First you need to run GreenCityClient
2. Create an order and partially paid it
3. The status of the order should be "Partially paid" | "Замовлення Oплачено" in Order History
4. That's it! ✔️

# PR Checklist Forms ✔️
